ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3698,7 +3698,39 @@ tests:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack
 - as: vsphere-ipi-ovn-dualstack-f28-ui
-  cron: 42 7 2 * *
+  cron: 32 9 29 * *
+  steps:
+    cluster_profile: vsphere
+    env:
+      E2E_RUN_TAGS: '@vsphere-ipi and @network-ovnkubernetes and not @fips and @console'
+      TAG_VERSION: '@4.14'
+    test:
+    - chain: openshift-e2e-test-qe-ui
+    workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack
+- as: vsphere-ipi-ovn-dualstack-privmaryv6-f28
+  cron: 28 20 15 * *
+  steps:
+    cluster_profile: vsphere
+    env:
+      E2E_RUN_TAGS: '@vsphere-ipi and @network-ovnkubernetes and not @fips'
+      IP_FAMILIES: DualStackIPv6Primary
+      TAG_VERSION: '@4.14'
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack
+- as: vsphere-ipi-ovn-dualstack-privmaryv6-f28-destructive
+  cron: 22 23 17 * *
+  steps:
+    cluster_profile: vsphere
+    env:
+      E2E_RUN_TAGS: '@vsphere-ipi and @network-ovnkubernetes and not @fips'
+      IP_FAMILIES: DualStackIPv6Primary
+      TAG_VERSION: '@4.14'
+    test:
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack
+- as: vsphere-ipi-ovn-dualstack-privmaryv6-f28-ui
```

Comment:
@yanpzhan is it ready to run ui testing in such env?

Reply:
No, we don't need this new ui job.  We don't have the requirement to add a separate ui job for a new workflow.

Reply:
Have you tested UI with IPv6?

Reply:
@rbbratta We have executed UI cases on cluster with IPv6. The chains "openshift-e2e-test-qe" and "openshift-e2e-test-qe-destructive" will also run some UI cases according to matrix, I think the logic should be the same with other team's cases. For those jobs with "-ui" suffix, we added them for special purpose to cover more ui scenarios.

Reply:
vSphere DualStackIPv6Primary is a new platform to 4.14.  It was only enabled a few days ago, no one has even seen the console yet, it's only available via Prow.

We added `vsphere-ipi-ovn-dualstack-f28-ui` but `vsphere-ipi-ovn-dualstack-privmaryv6-f28-ui` is new and should have priority over `vsphere-ipi-ovn-dualstack-f28-ui`.

Anyway we can change the tests later.